common/content/statusline.js

```diff
@@ -130,8 +130,7 @@ const StatusLine = Module("statusline", {
         if ( typeof this.setVisibility.currVisibility == 'undefined' ) {
             this.setVisibility.currVisibility = true;
             this.setVisibility.prevVisibility = true;
-            this.setVisibility.contentSeparator
-                = highlight.get('ContentSeparator').value;
+            this.setVisibility.passiveVisibility = false;
```

Comment:
Ah, I see. I wonder if there's a way to get whether we're in fullscreen and use that for checking against rather than adding another moving part.